ports/samd/boards/samd51x19a.ld

```diff
@@ -13,5 +13,8 @@ MEMORY
 _estack = ORIGIN(RAM) + LENGTH(RAM) - 8;
 _sstack = _estack - 16K;
 
+_oflash_fs = ORIGIN(FLASH) + 384K - 16K;
+_sflash_fs = LENGTH(FLASH) - 384K + 16K - 1;
```

Comment:
IMO these names are confusing, because "s" as a prefix usually means "start" in linker variables.  I suggest following stm32 and using `_micropy_hw_internal_flash_storage_start` and `_micropy_hw_internal_flash_storage_end` (or `_micropy_hw_internal_flash_storage_length`).

Reply:
Thanks for starting the review.
These names were existing before (committed  2019) and I did not change these. Obviously everything can be changed.

Reply:
Ah, OK.  In that case leave them as-is, we can change them later if needed.